webapp/src/styles.css

```
@@ -1,353 +1,577 @@
+    white 18%
+  );
+  --line: color-mix(in srgb, white 11%, transparent);
+  --line-strong: color-mix(in srgb, white 19%, transparent);
+  --text: #f3f1ea;
+  --muted: #9faaa2;
+  --accent: var(--accent-400);
+  --accent-soft: color-mix(in srgb, var(--accent-500) 18%, transparent);
+  --danger: #ef9a9a;
+  --shadow: 0 1.5rem 4rem color-mix(in srgb, var(--bg) 76%, transparent);
+  --sidebar-width: 15.5rem;
   color-scheme: dark;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)